Sources/TABResourceLoader/Protocols/Resource types/NetworkResourceType.swift

```diff
@@ -53,8 +53,11 @@ public protocol NetworkResourceType {
   /// The HTTP header fields used to fetch this resource
   var httpHeaderFields: [String: String]? { get }
 
-  /// The HTTP body as JSON used to fetch this resource
-  var jsonBody: Any? { get }
+  /// The HTTP body used to fetch this resource
+  var body: Any? { get }
```

Comment:
this isn't needed by this protocol. it's only used by the protocols that inherit from this protocol. i suggest declaring it specifically in each of the other protocols, which also means you can name it more appropriately for each of the protocols that actually uses it.

it could lead to confusion. e.g. if you create a concrete conformer of NetworkResourceType (that does _not_ conform to either NetworkJSONResourceType or NetworkPropertyListDecodableResourceType), then setting the `body` property does nothing.